inspire_disambiguation/core/ml/models.py

```diff
@@ -635,12 +624,14 @@ def load_data(self, signatures, input_clusters):
         self.X = np.empty((len(signatures_by_uuid), 1), dtype=np.object)
         self.y = -np.ones(len(self.X), dtype=np.int)
 
-        for i, cluster in enumerate(input_clusters):
+        i = 0
```

Comment:
why did you change this?

Reply:
I did the change to enumerate but for some reason it didn't work correctly with it - there is this continue statement in loop which should increase counter value and with enumerate it does it, what cause errors.